charts/fdb-operator/templates/manager/deployment.yaml

```diff
@@ -20,27 +21,43 @@ spec:
       labels:
         {{- include "chart.labels" . | nindent 8 }}
     spec:
+      securityContext:
+        runAsUser: 40590
```

Comment:
Any reason to change the default UID?

Reply:
Taking the UID above 10000 means that kube-score can validate the **other** components of the securityContext, without complaining about the UID. Apparently UIDs < 10000 are less secure - probably because they could accidentally belong to a system user ;)

Reply:
I would rather vote to leave the old UID I don't see any benefit in changing it and I don't see how UIDs < 10000 should be "less secure". We should rather configure Kube-score to ignore this check: https://github.com/zegl/kube-score/blob/master/README_SECURITYCONTEXT.md#removal-timeline-of-container-security-context

Reply:
Nice, I didn't know about the improvements to kube-score you linked above. I've updated accordingly, and reset the UID back to default.